src/Settings/KatapultSettings.php

```diff
@@ -7,10 +7,12 @@
 class KatapultSettings extends AppSettings
 {
     public string $public_key;
+    public string $private_key;
     public string $environment;
 
     protected array $rules = [
         'public_key' => ['required'],
+        'private_key' => ['required'],
         'environment' => ['required'],
     ];
 
```

Comment:
By making the setting encrypted in the migration we should also tell this file about it.

```suggestion

    public static function encrypted(): array
    {
        return ['private_key'];
    }

```